setup_requirements.py

```diff
@@ -103,7 +103,9 @@
         'pgtest==1.1.0',
         'sqlalchemy-diff==0.1.3',
         'coverage==4.5.1',
-        'codecov==2.0.15'
+        'codecov==2.0.15',
+        'pytest-concurrent==0.1.2',
+        'futures==3.2.0',
```

Comment:
```suggestion
        'futures; python_version == "2.7"',
```

And remove `pytest-concurrent` I don't think it is necessary

Reply:
Fixed.